packages/nvidia-k8s-device-plugin/nvidia-k8s-device-plugin-tmpfiles.conf

```diff
@@ -0,0 +1,2 @@
+d /etc/nvidia-k8s-device-plugin - - - - -
```

Comment:
I believe you could similarly drop the tmpfiles requirement here since both files are templates.

Reply:
deleted as suggested.